tensorflow/lite/toco/import_tensorflow_test.cc

```diff
@@ -385,6 +384,127 @@ std::vector<std::pair<tensorflow::DataType, ArrayDataType>> UnaryTestTypes() {
           {DT_INT64, ArrayDataType::kInt64}};
 }
 
+class TensorContentTest : public ::testing::Test {
+ public:
+  template <ArrayDataType T>
+  std::vector<DataType<T>> ImportAndGetData(const NodeDef& node) {
+    Model model;
+    auto status = ImportNode(node, &model);
+    CHECK(status.ok()) << status.error_message();
+    const auto& array = model.GetArray("Node1");
```

Comment:
Perhaps a more meaning full name than just ```array```?

Reply:
I have updated the same with nodearray